### PR TITLE
Support config the host keys for zuul log server known_hosts option

### PR DESCRIPTION
**Pull request by liusheng**
Related-Bug: theopenlab/openlab#217

### CODE REVIEW

File: playbooks/zuul-log-server.yaml [hunk 1]
```diff
@@ -23,6 +23,22 @@
         mode: 0755
         state: directory
 
+    - name: Set host private ecdsa key
+      when: config_host_ecdsa_key|default(true)|bool
+      copy:
+        content: "{{ known_host_key_priv }}"
+        dest: /etc/ssh/ssh_host_ecdsa_key
+        mode: 0600
```
Comment (wangxiyuan) [inline playbooks/zuul-log-server.yaml:31]: Double check: is this for `root` user, not `zuul` user?

File: playbooks/zuul-log-server.yaml [hunk 2]
```diff
@@ -23,6 +23,22 @@
...
+
+    - name: Set host public ecdsa key
+      when: config_host_ecdsa_key|default(true)|bool
+      shell: |
+        new_pub_key=`cat /etc/ssh/ssh_host_ecdsa_key.pub | awk '{print $1, "{{ known_host_key_priv }}", $3}'`
+        echo $new_pub_key > /etc/ssh/ssh_host_ecdsa_key.pub
+        chmod 0644 /etc/ssh/ssh_host_ecdsa_key.pub
```
Comment (wangxiyuan) [inline playbooks/zuul-log-server.yaml:38]: ditto

**Review by wangxiyuan — COMMENTED** [review]
LGMT, just a question

File: playbooks/zuul-log-server.yaml (hunk 1 above)
Comment (liusheng) [reply playbooks/zuul-log-server.yaml:31]: all the files under `/etc/ssh/` are root:root owned

File: playbooks/zuul-log-server.yaml (hunk 2 above)
Comment (liusheng) [reply playbooks/zuul-log-server.yaml:38]: ditto

**Review by kiwik — APPROVED** [review]
LGTM, thank you.

**Review by wangxiyuan — APPROVED** [review]
lgtm

**Pull request merged**